src/Bridge/Symfony/Bundle/DependencyInjection/Compiler/HandleDeprecatedPass.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Bridge\Symfony\Bundle\DependencyInjection\Compiler;
+
+use Symfony\Component\Config\Definition\BaseNode;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+final class HandleDeprecatedPass implements CompilerPassInterface
```

Comment:
DeprecateMercurePublisherPass

+ comment on the class that it's to handle a deprecation on the `setDeprecated` method